index.js

```diff
@@ -68,7 +68,26 @@ module.exports = function (str) {
   } else if ((match = str.match(new RegExp(regexps['re1'], 'i')))) {
     let days = match[1];
     let period = match[2];
+    if (period === 'year') {
+      let year = new Date().getFullYear() - days;
+      if (year < 0) {
+        return 'Before B.C.';
+      } else {
+        return [true, new Date(new Date().setFullYear(year))];
```

Comment:
Moving away from this format. Use this instead https://github.com/icyflame/relative-date-reverse/commit/68757bc655f21c00a7b166f384dfaa084a252421